main/webapp/modules/core/langs/translation-en.json

```diff
@@ -541,6 +541,8 @@
     "core-views/judg-hist": "Judgment action timestamp",
     "core-views/hist-entries": "$1: judgment action timestamp",
     "core-views/actions": "Actions",
+    "core-views/Add column with URLs of matched entities":"Add column with URLs of matched entities",
+    "core-views/Add column with URLs of matched entities2":"Add column with URLs of matched entities for all current filtered rows",
```

Comment:
The keys of the messages could be improved:
* as you can see in the neighboring messages, we have a convention to write the keys in lowercase and with dashes. I would follow this convention for the sake of uniformity (although the convention itself is debatable!)
* rather than adding `2` as suffix to differentiate the message for the tooltip, you could use something more explicit, like `-tooltip`

Reply:
done!